packages/expo-modules-core/README.md

```diff
@@ -24,6 +24,75 @@ Run `npx pod-install` after installing the npm package.
 
 No additional set up necessary.
 
+# Importing native dependencies - autolinking
+
+React Native libraries come with platform-specific (native) code. This native code has to be linked into the project or
+configured properly. These actions require some modifications to the native project files. One of the steps that have
+to be done with the native configuration is to enable autolinking mechanism that takes care of including any supported
+module's native code into the project. Below we preset the configuration that should be manually applied to the
+project's native files.
```

Comment:
```suggestion
Many React Native libraries come with platform-specific (native) code. This native code has to be linked into the project and, in some cases, configured further. These actions require some modifications to the native project files. One of the steps that has to be done with the native configuration is to enable the autolinking mechanism that takes care of including any supported module's native code into the project. The following configuration is required:
```

no need to add newlines here. better to leave it without newlines so it wraps based on editor width.

also -- we probably want to move this to the docs when we start to roll it out. we can replace https://docs.expo.dev/bare/installing-unimodules/